test/python/algorithms/optimizers/utils/test_learning_rate.py

```diff
@@ -36,7 +36,9 @@ def test_learning_rate(self):
         """
         constant_learning_rate_input = 0.01
         list_learning_rate_input = [0.01 * n for n in range(10)]
-        generator_learning_rate_input = lambda: (el for el in list_learning_rate_input)
+
+        def generator_learning_rate_input():
+            return (el for el in list_learning_rate_input)
```

Comment:
(if we disable the "don't assign lambdas" rule, we could revert this)

Reply:
The don't assign lambdas rule has been reverted. In any case I kind of like
```python
def generator_learning_rate_input(): return (el for el in list_learning_rate_input)
```